hello-ironhack-react-app/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+// import logo from './logo.svg';
+import './App.css';
+
+// import data (infos) to show in the InfoList
```

Comment:
comments are useful for complex constructions. Notes can be made elsewhere.
If your code is clear enough and in line with conventions, you wouldn't even need to write comments. 

Please avoid overly obvious comments in the future